unified-runtime/source/adapters/level_zero/v2/queue_batched.hpp

```diff
@@ -0,0 +1,442 @@
+//===--------------- queue_batched.hpp - Level Zero Adapter ---------------===//
+//
+// Copyright (C) 2025 Intel Corporation
+//
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM
+// Exceptions. See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "../common.hpp"
+#include "../device.hpp"
+
+#include "command_list_cache.hpp"
+#include "common/ur_ref_count.hpp"
+#include "context.hpp"
+#include "event.hpp"
+#include "event_pool_cache.hpp"
+#include "memory.hpp"
+#include "queue_api.hpp"
+
+#include "ur/ur.hpp"
+
+#include "command_buffer.hpp"
+#include "command_list_manager.hpp"
+#include "lockable.hpp"
+#include "queue_immediate_in_order.hpp"
+#include "ur_api.h"
+#include "ze_api.h"
+
+/*
+Batched queues enable submission of operations to the driver in batches,
+therefore reducing the overhead of submitting every single operation
+individually. Similarly to command buffers in L0v2, they use regular command
+lists (later referenced as 'batches'). Operations enqueued on regular command
+lists are not executed immediately, but only after enqueueing the regular
+command list on an immediate command list. However, in contrast to command
+buffers, batched queues also handle submission of batches (regular command
+lists) instead of only collecting enqueued operations, by using an internal
+immediate command list. Command lists are managed by a batch_manager inside a
+batched queue.
+
+Batched queues can be enabled by setting UR_QUEUE_FLAG_SUBMISSION_BATCHED in
+ur_queue_flags_t or globally, through the environment variable
+UR_L0_FORCE_BATCHED=1.
+*/
+
+namespace v2 {
+
+struct batch_manager {
```

Comment:
You use three different styles of multi-line comments in this struct. I think the most commonly used style in the adapter codebase is:
```
//
//
//
```
But if you want to use block-style comments, do:
```
/*
 * ...
 * ...
 */
```